internal/config/config.go

```diff
@@ -25,14 +25,15 @@ type Configuration struct {
 	Verbosity   int
 	Debug       bool
 	DebugListen string
-	API         api.Config
-	Event       event.Config
-	Database    db.Config
-	Temporal    temporal.Config
-	Watcher     watcher.Config
-	Storage     storage.Config
-	Upload      upload.Config
-	A3m         a3m.Config
+
+	A3m      a3m.Config
+	API      api.Config
+	Database db.Config
+	Event    event.Config
+	Storage  storage.Config
```

Comment:
Why reorder here?

Reply:
I was writing the tests and and find it easier to check for completeness in an ordered list than an unordered one.